.bazelci/presubmit.yml

```diff
@@ -0,0 +1,9 @@
+---
+platforms:
+  ubuntu1604:
+    shell_commands:
+      - sh setup_workspace.sh
+    build_targets:
+      - "//test_suite:all"
+    test_targets:
+      - "//test_suite:all"
```

Comment:
I suggest `//...` for both build_targets and test_targets.
This will automatically pick up any new test we add.